Winapp2.ini

```diff
@@ -1559,8 +1535,7 @@ FileKey1=%AppData%\Adobe\Photoshop Album\*.*|Logse*.txt
 [Adobe Photoshop CS2 More*]
 LangSecRef=3021
 Detect=HKCU\Software\Adobe\Photoshop\9.0
-DetectFile1=%AppData%\Adobe\CameraRaw\Cache
-DetectFile2=%AppData%\Adobe\Bridge\Cache
+DetectFile1=%AppData%\Adobe\*\Cache
```

Comment:
Do wildcards work in Detectfile?

Reply:
Yes. They didn't used to but I tested with a recent CCleaner and they seemed to be, even with nested wildcards